probes/python_probe.py

```diff
@@ -0,0 +1,21 @@
+import os
+
```

Comment:
One could enclose the function body with a try-except clause to let the function fail silently in case "os.utime(filename)" won't work for whatever reasons.